ignite/handlers/wandb_logger.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *args: Any, **kwargs: Any):
                 "You man install wandb with the command:\n pip install wandb\n"
             )
         if kwargs.get("init", True):
+            kwargs.pop('init', None)
```

Comment:
Black formatting requires double quotes instead of single quotes, please change it

Reply:
ok